Formula/trzsz.rb

```diff
@@ -0,0 +1,39 @@
+class Trzsz < Formula
+  include Language::Python::Virtualenv
+
+  desc "Simple file transfer tools, similar to lrzsz (rz/sz), and compatible with tmux"
+  homepage "https://trzsz.github.io"
+  url "https://files.pythonhosted.org/packages/50/01/b8679a72c0604c964a6c4f46f100fa707ecad97c34f61e58334f764e9cca/trzsz-0.3.1.tar.gz"
+  sha256 "97b008cd5441583e9090c9078854e90b3cdfe2fd95a1cfb717efee1b935a7737"
+  license "MIT"
+
+  depends_on "python@3.9"
```

Comment:
Can we use `python@3.10`?

Reply:
Yes, we can. Any `python3` will be ok. Should I change it to `python@3.10` ?

Reply:
Yes please.